js/coinex.js

```diff
@@ -2112,6 +2114,250 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async fetchPositions (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        let market = undefined;
+        if (symbols !== undefined) {
+            market = this.market (symbols);
```

Comment:
The `symbols` argument is an array of symbols here, so we can't treat it as a single symbol.

Reply:
For the same reason we usually do `return this.filterByArray (positions, 'symbol', symbols, false)` at the end of methods accepting an array of symbols.